freqtrade/arguments.py

```diff
@@ -184,11 +182,12 @@ def optimizer_shared_options(parser: argparse.ArgumentParser) -> None:
             dest='refresh_pairs',
         )
 
-    @staticmethod
-    def backtesting_options(parser: argparse.ArgumentParser) -> None:
+    def backtesting_options(self, subparser: argparse.ArgumentParser = None) -> None:
         """
         Parses given arguments for Backtesting module.
         """
+        parser = subparser or self.parser
```

Comment:
an alternative way (for the future, maybe) could be to pass parser always to the function so we can save these comparisons (it's not like they are called 100ds of times per cycle ...).